### PR TITLE
Remove ineligible disabilities

### PR DESCRIPTION
**Pull request by cvalarida**
Nothing new to show here. Just filtered out disabilities without rated disabilities.

### CODE REVIEW

**Comment by cvalarida** [comment]
@goldenmeanie @peggygannon Should we have any messaging on the disabilities selection screen saying that we may have pulled some conditions out?

@djmunoz1 I'd like to log the disabilities we pull out if they have a `decision_code` of `NOTSVCCONNCTED` (or whatever it is; it's not recorded in the swagger). Can I log some or all of each condition to sentry without worrying about PHI concerns?

**Comment by goldenmeanie** [comment]
It seems like we could follow the list with a note like:

> This list only includes disabilities that you have received a rating for. It will not show disabilities from claims in progress.

**Comment by cvalarida** [comment]
@peggygannon Does this content look good?
@goldenmeanie What do you think of the spacing between the last disability and the descriptive text?
image

**Comment by goldenmeanie** [comment]
Good eye Chris, spacing is a little tight

**Comment by peggygannon** [comment]
@cvalarida thank you...  just a few editorial tweaks from me:

**Please note:** This list only includes disabilities that we've already rated. It doesn't include any disabilities from claims that are in progress.

**Comment by cvalarida** [comment]
@goldenmeanie @peggygannon How's this look?
image

The spacing is still not even with the top, but it's an even (odd...) `1em`. I can match if that's what you'd prefer, @goldenmeanie.

**Comment by peggygannon** [comment]
@cvalarida LGTM! thank you

File: src/applications/disability-benefits/526EZ/helpers.jsx [hunk 1]
```diff
@@ -103,16 +103,14 @@ export function validateDisability(disability) {
...
-    }
-    return newDisability;
-  });
+  return disabilities
+    // We want to remove disabilities without a rating, but 0 counts as a valid rating
+    // TODO: Log the disabilities if they're not service connected
+    // Unfortunately, we don't have decisionCode in the schema, so it's stripped out by the time
+    //  it gets here and we can't tell whether it is service connected or not. This happens in
+    //  the api
+    .filter(disability => disability.ratingPercentage || disability.ratingPercentage === 0)
```
Comment (markgreenburg) [inline src/applications/disability-benefits/526EZ/helpers.jsx]: Much nicer :)

**Review by markgreenburg — APPROVED** [review]
LGTM! I think we're also filtering disabilities in a couple other places, e.g. the confirmation page, so might want to remove duplicate / value-less code there.

**Comment by cvalarida** [comment]
**TODO:** Log the decision code and text when the disability is not rated and the decision code is not `NONSVCONNCTD` or whatever.

**Comment by goldenmeanie** [comment]
Note spacing looks comfortable relative to neighboring elements @cvalarida. Thanks!

**Pull request merged**